v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -77,7 +79,41 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
 };
 
 const processTrackEvent = async (metadata, message, destination) => {
-  const payload = constructPayload(message, trackMapping);
+  let flag = 0;
+  const { Config } = destination;
+  const { event } = message;
+  const { listOfConversions } = Config;
+  for (let i = 0; i < listOfConversions.length; i += 1) {
+    if (listOfConversions[i].conversions === event) {
+      flag = 1;
+      break;
+    }
+  }
+  if (event === undefined || event === "" || flag === 0) {
+    throw new ErrorBuilder()
+      .setMessage(
+        `[Google_adwords_enhanced_conversions]:: Conversion named ${event} is not exist in rudderstack dashboard`
+      )
+      .setStatus(400)
+      .build();
+  }
+  const { requireHash } = destination.Config;
+  let updatedMapping = JSON.parse(JSON.stringify(trackMapping));
```

Comment:
Ref: https://stackoverflow.com/questions/122102/what-is-the-most-efficient-way-to-deep-clone-an-object-in-javascript

Lets use loadash cloneDeep